nft-v2/src/main.rs

```diff
@@ -678,6 +678,12 @@ fn mint(
     })
 }
 
+#[update(name = "burn", guard = "is_canister_owner")]
```

Comment:
We should still include the example implementation of burn

Reply:
yeah, we can address this in another PR as it should be simple for now
otherwise, we have to write the integration test for it, not worth at the moment